src/commands/appSettings/confirmOverwriteSettings.ts

```diff
@@ -0,0 +1,80 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import { DialogResponses } from "vscode-azureextensionui";
+import { ext } from "../../extensionVariables";
+
+export async function confirmOverwriteSettings(sourceSettings: { [key: string]: string }, destinationSettings: { [key: string]: string }, destinationName: string): Promise<void> {
```

Comment:
This function feels like too much code to just copy from functions. I think it should be in shared package